docs/index.rst

```diff
@@ -84,12 +93,12 @@ We can set Bob's :code:`caffeineChoice`, his default is coffee
 .. exec::
 
     import properties #hide
-    class CoffeeProfile(properties.PropertyClass):#hide
+    class CoffeeProfile(properties.HasProperties):#hide
         import properties #hide
         myName = properties.String('What should I call you?', required=True)#hide
```

Comment:
Can we please make this example pep8-ified.

`Coffee.name` and `haz_enough_coffee` etc.